src/js/event-target.js

```diff
@@ -141,6 +141,8 @@ EventTarget.prototype.trigger = function(event) {
   if (typeof event === 'string') {
     event = {type};
   }
+
+  event = Object.assign({target: this}, event);
```

Comment:
I'm a bit worried about what changing the default target could do

Reply:
yeah agreed, after testing some more I think that we have to remove this.